### PR TITLE
TF-1434 Implement `autofillHints` for login view

### PR DESCRIPTION
**Pull request by dab246**
### Issues

#1434 

### Wait merge

- https://github.com/AbdulRahmanAlHamali/flutter_typeahead/pull/466

### Desc

- Minimum SDK version supports `autofill`: `Android >= 26 | iOS >= 10.0`
- Configuration `autofill` on mobile: 
    + **Android**: Go to `Settings -> System -> Languages & input -> Autofill service`. 
    + **iOS**: Go to `Settings -> Password, turn on "Autofill Passwords"`.
    + See more at [here](https://api.flutter.dev/flutter/material/TextField/autofillHints.html)

### Demo

- **Android: version 29**

https://user-images.githubusercontent.com/80730648/221745083-dc9808e7-e73a-45cb-97dd-e66090905ee8.mov

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -216,6 +213,11 @@ dependencies:
 
   focused_menu_custom: 1.2.0
 
+  flutter_typeahead:
+    git:
+      url: https://github.com/dab246/flutter_typeahead.git
```
Comment (hoangdat) [inline pubspec.yaml:218]: what happen with it?

Comment (dab246) [reply pubspec.yaml:218]: Well, I wanted to add support for the `autofillhints` property of `TextField` so I created a new branch to handle it.

Comment (hoangdat) [reply pubspec.yaml:218]: please contribute to upstream :)

Comment (dab246) [reply pubspec.yaml:218]: Very difficult, currently their repository is supporting flutter `3.7.x` so we need to wait for `Tmail` to be upgraded to fluttter version `3.7.7.5`

**Review by nqhhdev — APPROVED** [review]

**Pull request merged**